src/main/java/com/example/demo/model/Book.java

```diff
@@ -34,4 +41,12 @@ public class Book {
     private String coverImage;
     @Column(nullable = false, columnDefinition = "TINYINT(1) DEFAULT 0")
     private boolean isDeleted = false;
+    @ManyToMany(fetch = FetchType.LAZY)
```

Comment:
```suggestion
    @ManyToMany
```
It's default fetch type